cmake/FindTBB.cmake

```diff
@@ -6,9 +6,22 @@ find_library(TBB_LIB NAMES tbb
   HINTS /usr/local/lib /usr/lib /usr/lib/x86_64-linux-gnu
   DOC "TBB libraries")
 
-# if(GTSAM_LIB AND GTSAM_UNSTABLE_LIB AND TBB_LIB)
-if(TBB_LIB)
-  set(TBB_LIBRARIES ${TBB_LIB})
+find_library(TBB_MALLOC_LIB NAMES tbbmalloc
+  HINTS /usr/local/lib /usr/lib
+  DOC "TBB malloc libraries")
+
```

Comment:
I got the following error when `BUILD_EXAMPLES` and `BUILD_BENCHMARKS` were both `ON`.

```
CMake Error at cmake/FindTBB.cmake:13 (add_library):
   add_library cannot create imported target "TBB::tbb" because another target
   with the same name already exists.
```

It's caused by my bad CMakeLists that calls `find_package(TBB)` multiple times, and I will fix it later. For now, could you add the following here as a workaround?

```cmake
if(TARGET TBB::tbb)
  return()
endif()
```